be/src/olap/rowset/rowset_meta.h

```diff
@@ -0,0 +1,173 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#ifndef DORIS_BE_SRC_OLAP_ROWSET_ROWSET_META_H
+#define DORIS_BE_SRC_OLAP_ROWSET_ROWSET_META_H
+
+#include "gen_cpp/olap_file.pb.h"
+
+#include <memory>
+#include <vector>
+
+#include "olap/new_status.h"
+
+namespace doris {
+
+class RowsetMeta {
+public:
+    virtual void init(const RowsetMetaPb& rowset_meta_pb) {
```

Comment:
RowsetMeta use deserialize interface to convert bytes to RowsetMetaPB. 
Only RowsetMeta can know it is stored as pb or another format.

Reply:
ok